ext/strings.go

```diff
@@ -298,12 +299,7 @@ func (sl stringLib) CompileOptions() []cel.EnvOption {
 				},
 			}
 		}
-		formatOverload = cel.MemberOverload("string_format", []*cel.Type{cel.StringType, cel.ListType(cel.DynType)}, cel.StringType,
-			cel.FunctionBinding(func(args ...ref.Val) ref.Val {
-				s := args[0].(types.String).Value().(string)
-				formatArgs := args[1].(traits.Lister)
-				return stringOrError(stringFormatWithLocale(s, formatArgs, sl.locale))
-			}))
+		formatLocale = sl.locale
```

Comment:
This is probably me being paranoid, but could you update the local `formatLocale` immediately after declaration?

Reply:
Gotcha, I'll swap the order of the `if sl.locale != ""` block and the `formatOverload` declaration.